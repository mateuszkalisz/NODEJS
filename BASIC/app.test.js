const request = require('supertest');
const {app} = require('./app');

it('works', async ()=>{
    const response = await request(app).get('/');
    
    // expect(response).toMatchSnapshot();
    expect(response.status).toEqual(200);
    expect(response.text).toEqual('Hello World');

});