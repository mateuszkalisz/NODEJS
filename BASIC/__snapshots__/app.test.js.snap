// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
Object {
  "header": Object {
    "connection": "close",
    "content-length": "5",
    "content-type": "text/html; charset=utf-8",
    "date": "Tue, 11 Feb 2020 19:59:28 GMT",
    "etag": "W/\\"5-M9myltz/OZ1KnTI273DXvFZqJqM\\"",
  },
  "req": Object {
    "data": undefined,
    "headers": Object {
      "user-agent": "node-superagent/3.8.3",
    },
    "method": "GET",
    "url": "http://127.0.0.1:64750/",
  },
  "status": 200,
  "text": "WITAJ",
}
`;
